cli/tests/test_git_fetch.rs

```diff
@@ -297,6 +297,114 @@ fn test_git_fetch_multiple_remotes(subprocess: bool) {
     }
 }
 
+#[test_case(false; "use git2 for remote calls")]
+#[test_case(true; "spawn a git subprocess for remote calls")]
+fn test_git_fetch_with_glob(subprocess: bool) {
+    let test_env = TestEnvironment::default();
+    if !subprocess {
+        test_env.add_config("git.subprocess = false");
+    }
+    test_env.run_jj_in(".", ["git", "init", "repo"]).success();
+    let repo_path = test_env.env_root().join("repo");
+    add_git_remote(&test_env, &repo_path, "rem1");
+    add_git_remote(&test_env, &repo_path, "rem2");
+
+    let output = test_env
+        .run_jj_in(&repo_path, ["git", "fetch", "--remote", "glob:*"])
+        .success();
```

Comment:
nit: `.success()` can be omitted when the output is snapshotted. (applies to the other tests.)

Reply:
Dropped `.success()` call from snapshotted outputs